### PR TITLE
[MM-10858] Update websocket actions

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Update websocket actions to dispatch roles when channel member role is changed

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-10858

Awaiting https://github.com/mattermost/mattermost-server/pull/13305

### CODE REVIEW

**Comment by hahmadia** [comment]
@saturninoabril Added unit test as requested.

File: src/actions/websocket.test.js [hunk 1]
```diff
@@ -321,6 +326,23 @@ describe('Actions.Websocket', () => {
...
+            }
+        }));
+        expect(mockLoadRolesIfNeeded).toHaveBeenCalledTimes(1);
+        expect(mockLoadRolesIfNeeded).toHaveBeenCalledWith(channelMember.roles.split(' '));
+        store.subscribe(() => {
+            const state = store.getState();
+            expect(state.entities.channels.membersInChannel[channelMember.channel_id][channelMember.user_id].roles).toEqual("channel_user channel_admin");
+        });
+    });
+
```
Comment (saturninoabril) [inline src/actions/websocket.test.js]: 💯

**Comment by amyblais** [comment]
@hahmadia Can this be moved to v5.20? v5.19 is a quality release and we should only include bug fixes and no features.

**Comment by hahmadia** [comment]
@amyblais The reason I put this for 5.19 is because it's part of https://mattermost.atlassian.net/browse/MM-10858 which seemed like a bug to me. If it isn't I will gladly move all the tickets relating to MM-10858 to 5.20. Thoughts?

**Pull request merged**